### DISCUSSION TITLE
"Your main branch isn't protected" on the branch list is not dismissable

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#21205] [Other Feature Feedback, Questions, & Ideas] [answered]
It's not possible for me to dismiss this notice when I click on the "Dismiss" button:
![image](https://user-images.githubusercontent.com/6032823/180668561-1be6713e-5256-4297-84f1-ec803f4c676b.png)
Link to the affected page: https://github.com/jack1142/cibuildwheel/branches

I am using Chrome 103 on Ubuntu 22.04 and this is reproducible without any extensions installed.

**Comment by radar-blip** [comment]
Just click on "Protect this branch" and dont select anything.

**Reply by Jackenmen** [reply]
I'm aware of how to work around this but what I want is to be able to dismiss it as can be done with all the other notices that GitHub has for the lack of branch protection. Personally, I think it nags you way too much on all of your repositories but regardless of what I think about it, the dismiss button is clearly broken here.

**Comment by radar-blip** [comment, marked as answer]
If it is in fact a broken link, you can file a bug report to help get it fixed (the sooner the better). 
You can file a bug report here: 
https://support.github.com/contact/bug-report

**Reply by Jackenmen** [reply]
Thanks, I sent a ticket.

**Reply by radar-blip** [reply]
You're welcome.
